libvast/test/format/json.cpp

```diff
@@ -0,0 +1,79 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/format/json.hpp"
+
+#define SUITE format
+
+#include "vast/test/test.hpp"
+
+#include "vast/test/fixtures/actor_system.hpp"
+#include "vast/test/fixtures/events.hpp"
+
+#include "vast/concept/parseable/to.hpp"
+#include "vast/event.hpp"
+
+using namespace vast;
+using namespace std::string_literals;
+
+namespace {
+
+auto http = record_type{{"ts", vast::timestamp_type{}},
+                        {"uid", vast::string_type{}},
+                        {"id.orig_h", vast::address_type{}},
+                        {"id.orig_p", vast::port_type{}},
+                        {"id.resp_h", vast::address_type{}},
+                        {"id.resp_p", vast::port_type{}},
+                        {"trans_depth", vast::count_type{}},
+                        {"method", vast::string_type{}},
+                        {"host", vast::string_type{}},
+                        {"uri", vast::string_type{}},
+                        {"version", vast::string_type{}},
+                        {"user_agent", vast::string_type{}},
+                        {"request_body_len", vast::count_type{}},
+                        {"response_body_len", vast::count_type{}},
+                        {"status_code", vast::count_type{}},
+                        {"status_msg", vast::string_type{}},
+                        {"tags", vast::vector_type{vast::string_type{}}},
+                        {"resp_fuids", vast::vector_type{vast::string_type{}}}}
```

Comment:
Do we need the `vast::` prefixes here, given that you have `using namespace vast` above?